### ISSUE TITLE
[Feature Request]: when will the electron will support windows 11 new feature? like [snap flyout] [rounded corners]

### ISSUE THREAD

**Issue by zhangzhui** [electron/electron#30468]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

when will the electron will support windows 11 new feature? like [snap flyout] [rounded corners]

### Proposed Solution

is there someone tell me where is the roadmap of electron?

### Alternatives Considered

[empty]

### Additional Information

[empty]

**Comment by miniak** [comment]
@zhangzhui are they not supported?

**Comment by zhangzhui** [comment]
i can't find the feature in electron release page
will you like to tell me where i can find it?

**Comment by miniak** [comment]
@Levminer good point, those are rendered by Chromium. They don't seem to be rounded even in Chrome 93 running on Windows 11
